### PR TITLE
guess the extension from content_type for inline attachment

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

**Comment by nicpottier** [comment]
Unit test?

File: temba/ivr/tests.py [hunk 1]
```diff
@@ -583,3 +584,29 @@ def test_incoming_start(self):
...
+
+            with patch('temba.orgs.models.Org.save_media') as mock_save_media:
+                mock_save_media.return_value = 'SAVED'
+
+                output = twilio_client.download_media('http://api.twilio.com/ASID/Media/SID')
+                self.assertIsNotNone(output)
+                self.assertEqual(output, 'text/x-vcard:SAVED')
+                # saved_media was called with a file as first argument and the guessed extension as second argument
+                self.assertIsInstance(mock_save_media.call_args_list[0][0][0], File)
+                self.assertEqual(mock_save_media.call_args_list[0][0][1], 'vcf')
```
Comment (nicpottier) [inline temba/ivr/tests.py]: Would be good to have a full stack test that starts from an incoming VCF attachment on the TwilioHandler and testing that the Msg is created appropriately.

Comment (norkans7) [inline temba/ivr/tests.py]: the test is added in https://github.com/nyaruka/rapidpro/pull/759/files#diff-028bbc16f694753d946ca5d67a2737a4R4753

**Comment by ycleptkellan** [comment]
Good to go?

**Comment by norkans7** [comment]
Yes, let's wait for one more review from @nicpottier

**Comment by nicpottier** [comment]
nice!

**Pull request merged**